docs/implementation-references/manufacturer-guide.md

```diff
@@ -0,0 +1,59 @@
+## Introduction
+FIDO Device Onboard (FDO) enhances the out-of-the-box setup and provisioning experience for connected IoT devices. The FIDO Device Onboard project provides the manufacturer component sample for manufacturers (OEMs, ODMs, or 3rd party integrators) to enable FDO on their devices. This document details how a manufacturer can use this tool to produce an FDO-enabled device. 
+
+### Terminology
+Refer to the FIDO Device Onboard [Reference page](../../reference.md).
```

Comment:
This might not render properly because of the location of reference.md in GitHub.IO.

We can fix this later after it has been rendered.